src/main/scala/cromwell/binding/Call.scala

```diff
@@ -56,15 +58,27 @@ case class Call(alias: Option[String], task: Task, inputMappings: Map[String, Wd
    * of a prerequisite call), that prerequisite call will appear only once in the output.
    */
   def prerequisiteCalls(): Set[Call] = {
-    val allCalls = parent.get.asInstanceOf[Workflow].calls
-
-    val prerequisiteCalls = for {
+    inputMappings.values.flatMap {expr =>
```

Comment:
I tried that, had to swap it to double flatMaps based on some Scala discussion I found on StackOverflow.  However since I migrated to double flatMaps, the code has changed so maybe it is compatible with a for comprehension.